MediaBrowser.Model/System/SystemInfo.cs

```diff
@@ -132,8 +114,6 @@ public SystemInfo()
         /// <value><c>true</c> if this instance has update available; otherwise, <c>false</c>.</value>
         public bool HasUpdateAvailable { get; set; }
 
-        public FFmpegLocation EncoderLocation { get; set; }
```

Comment:
Don't remove properties from API-exposed models. Mark them as deprecated and put a default value in it instead. This is to prevent clients using an older SDK/apiclient running into deserialization issues.

Reply:
Fixed